src/plugins/autologin/MQ2AutoLogin.cpp

```diff
@@ -122,6 +124,11 @@ class LoginProfileType : public MQ2Type
 			Dest.Int = record->characterLevel;
 			Dest.Type = mq::datatypes::pIntType;
 			return true;
+		case LoginProfileMembers::CustomCharacterIni:
+			strcpy_s(DataTypeTemp, record->customClientIni ? record->customClientIni->c_str() : "No Custom INI");
```

Comment:
Wouldn't this be better if it were empty vs a magic string that everybody would need to know means "empty"?

Reply:
I waffled on that a bit doing `/echo ${AutoLogin.Profile.CustomCharacterIni} and getting just an empty space seemed like it might be confusing, i figured instead of saying "none" or "null" or "" saying "No Custom INI" would be pretty clear.

that's just my rational, I'm not married to one or the other.

I also found it interesting it appears the code requires "eqclient" to be in the customini name "sic-eqclient.ini" for example, which wasn't obvious.

let me know if you would like the empty string

Reply:
Empty string seems like the universal standard when theres no value.